File: C++/Easy/WrongAnswer/redistribute-characters-to-make-all-strings-equal_9.cpp
class Solution {
public:
    bool makeEqual(vector<string>& words) {
        
        int n = words.size();
        int m = words[0].length();
        int sum = 0;
        for (int i = 0; i < n; i++) {
            sum += words[i].length();
        }
        if (sum % n != 0) return false;
        int avg = sum / n;
        for (int i = 0; i < n; i++) {
            if (words[i].length() != avg) return false;
        }
        return true;
    }
};

File: Rust/Easy/CompileError/redistribute-characters-to-make-all-strings-equal_9.rs
impl Solution {
    pub fn make_equal(words: Vec<String>) -> bool {
        
        let mut c = [0; 26];
        for s in words {
            for c in s.bytes() {
                c[c as usize - 'a' as usize] += 1;
            }
        }
        for i in 0..26 {
            if c[i] % words.len() != 0 {
                return false;
            }
        }
        true
    }
}